### PR TITLE
dcos-3103 Add post uninstall notes reporting

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: cli/tox.ini [hunk 1]
```diff
@@ -23,11 +23,11 @@ commands =
 
 [testenv:py27-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
 
 [testenv:py34-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
```
Comment (jsancio) [inline cli/tox.ini]: @mgummelt and @tamarrow, I know that you both talked about this but I found it very helpful while testing this change locally. What do you we think?

Comment (tamarrow) [inline cli/tox.ini]: I've always been for it :)

File: cli/tests/data/dcos.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [core]
 reporting = false
-timeout = 5
 email = "test@mail.com"
+timeout = 5
```
Comment (tamarrow) [inline cli/tests/data/dcos.toml]: why did we move this?

File: cli/tox.ini (hunk 1 above)
Comment (mgummelt) [inline cli/tox.ini]: this is temporary while I'm testing

File: dcos/package.py [hunk 3]
```diff
@@ -260,6 +263,17 @@ def is_match(app):
             logger.info(
                 'Found the following frameworks: {}'.format(framework_ids))
 
+            # Emit post uninstall notes
+            emitter.publish(
+                DefaultError(
+                    'Uninstalled package [{}] version [{}]'.format(
+                        package_json.get('name'),
+                        package_json.get('version'))))
```
Comment (tamarrow) [inline dcos/package.py]: We should probably put the `name`/`version` in an `if` or else the message wouldn't make sense

**Comment by tamarrow** [comment]
lgtm, just a few nits

File: cli/tox.ini (hunk 1 above)
Comment (mgummelt) [inline cli/tox.ini]: oh sorry.  I thought this was my PR.  It's helpful with local testing, but I'd rather see all test results from the async test run.

Up to you guys now though.

File: cli/tests/data/dcos.toml (hunk 2 above)
Comment (jsancio) [inline cli/tests/data/dcos.toml]: Our TOML library moves them around since it doesn't use an order dictionary.

File: dcos/package.py (hunk 3 above)
Comment (jsancio) [inline dcos/package.py]: They have to be set or else it if a bug. I'll fix the code.

File: cli/tox.ini (hunk 1 above)
Comment (jsancio) [inline cli/tox.ini]: Okay, I'll change it to `-x` since we should be running tests locally before pushing PRs.

**Comment by jsancio** [comment]
@tamarrow updated PR.

**Comment by tamarrow** [comment]
@jsancio :ship:

**Pull request merged**